build/x86_64_debian/Dockerfile

```diff
@@ -41,7 +18,7 @@ RUN apt-get -y update \
         make \
         ruby \
  # RVM GPG keys.
- && gpg2 --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB \
```

Comment:
Why not keep the `--keyserver` argument? It just makes this explicit, doesn't it? Plus, all of the other dockerfiles have this argument.

Reply:
You're right, the latest instructions on rvm.io don't contain the key server. +1 on the cleanup PR.

Reply:
See followup in https://github.com/GoogleCloudPlatform/google-fluentd/pull/253.